### PR TITLE
fix: MD022/blanks-around-headings/blanks-around-headers

### PR DESCRIPTION
**Pull request by nschonni**
Headings should be surrounded by blank lines

### CODE REVIEW

**Comment by XhmikosR** [comment]
Does remark-lint have a similar rule?

File: tests/scripts/CHANGELOG.fixture.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 will be supported actively until April 2017 and maintained until April 2018.
 
 <a id="4.4.4"></a>
+
```
Comment (XhmikosR) [inline tests/scripts/CHANGELOG.fixture.md:48]: We should exclude `tests/**/*.md` from this IMO.

**Comment by XhmikosR** [comment]
The English changes are fine. The translations are mostly because they are using HTML comments with the English text. So, it should be OK.

Also, I'm not sure if the test files should keep getting modified; they are historical files.

File: tests/scripts/CHANGELOG.fixture.md (hunk 1 above)
Comment (Trott) [reply tests/scripts/CHANGELOG.fixture.md:48]: Ideally, this file should probably be in a `fixtures` dir that all linting should ignore. But that doesn't have to happen in this PR necessarily.

**Comment by XhmikosR** [comment]
@nschonni does remark-lint have a rule for this?

**Comment by nschonni** [comment]
I think https://github.com/remarkjs/remark-lint/blob/master/doc/comparison-to-markdownlint.md is out of date, but it isn't a rule that is in the current baseline node preset. It's good because stuff like VS Code's markdown parser also trips up on rendering stuck headers. The upstream rule will need some cleanup because of the current stuck HTML comments used for generation

**Comment by XhmikosR** [comment]
We juts need a rule because this is going to happen again. My suggestion would be to not merge this until we have a check for it.

**Comment by wooorm** [comment]
Is [`remark-lint-no-missing-blank-lines`](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-no-missing-blank-lines#remark-lint-no-missing-blank-lines) what you’re looking for?

**Comment by XhmikosR** [comment]
Thanks @wooorm! @nschonni can you add this rule to this repo's rules?

**Comment by Chasdaspaz** [comment]
pull request

**Comment by Chasdaspaz** [comment]
pull request

**Comment by Chasdaspaz** [comment]
pull request

**Comment by XhmikosR** [comment]
This needs a rebase and the rule https://github.com/nodejs/nodejs.org/pull/2695#issuecomment-544114526 added

**Comment by XhmikosR** [comment]
@nschonni can you rebase this and add the remark-lint rule please?

**Pull request closed without merging**